src/Keboola/ElasticsearchWriter/Writer.php

```diff
@@ -60,6 +67,12 @@ public function loadFile(CsvFile $file, LoadOptions $options, $primaryIndex = nu
 	{
 		$csvHeader = $file->getHeader();
 
+		try {
```

Comment:
Ak vznikne pri vytvarani indexu nejaka chyba, tak sa to vyhodi ako warning, kedze index sa vie vytvorit aj automaticky pri prvom nahravani dát - len tam nebude mapovanie typov.

Je to tak z dovodu, ze my testujeme iba oproti 3 verziam ES, no v ES sa neustale nieco meni, tak aby pouzivatel neprisiel o moznost nahrat data iba preto, ze z nejakeho dovodu neprejde vytvorenie indexu.